crates/ark-contracts/arkchain/src/orderbook.cairo

```diff
@@ -37,18 +47,27 @@ mod orderbook_errors {
     const ORDER_INVALID_DATA: felt252 = 'OB: order invalid data';
     const ORDER_ALREADY_EXEC: felt252 = 'OB: order already executed';
     const ORDER_NOT_FOUND: felt252 = 'OB: order not found';
+    const ORDER_FULFILLED: felt252 = 'OB: order fullfiled';
+    const STATUS_NOT_FOUND: felt252 = 'OB: status not found';
```

Comment:
Status not found is not something that can happen. If the status is not one of the valid status, the order does not exist.